modules/swagger-codegen/src/main/resources/Java/apiInvoker.mustache

```diff
@@ -25,10 +25,30 @@ public class ApiInvoker {
   {{! wikia }}
   private static final String RESTRICTED_HEADERS_PROPERTY = "sun.net.http.allowRestrictedHeaders";
   {{! /wikia }}
+  public interface ClientFactory {
+    public Client getClient(String path);
+  }
+
+  public class DefaultClientFactory implements ClientFactory {
+    private Map<String, Client> hostMap = new HashMap<String, Client>();
+
+    protected Client createClient(String path) {
+      return ClientBuilder.newClient();
+    }
+
+    public Client getClient(String path) {
```

Comment:
Yeah, I'll change it although this name is confusing, because it stores values like "http://www.wikia.com/api/v1" so it's not just the host name. That's why I changed it in the first place, so my recommendation would be to change it from host to path/url in all places and have it consistent.